src/mcts/stoppers/factory.cc

```diff
@@ -114,6 +117,7 @@ void PopulateTimeManagementOptions(RunType for_what, OptionsParser* options) {
     options->Add<FloatOption>(kTimeMidpointMoveId, 1.0f, 100.0f) = 51.5f;
     options->Add<FloatOption>(kTimeSteepnessId, 1.0f, 100.0f) = 7.0f;
     options->Add<FloatOption>(kSpendSavedTimeId, 0.0f, 1.0f) = 1.0f;
+    options->Add<BoolOption>(kNodesAsPlayoutsId) = false;
 
     // Hide time curve options.
     options->HideOption(kTimeMidpointMoveId);
```

Comment:
Do you think it may make sense to hide the new option too, in order not to overwhelm the UCI settings page?

Reply:
I think it's probably better not to hide the new option, so that users can be informed. Otherwise I would expect very few people to know and use it.

Reply:
But why would more than a few people need it? I think that the only scenario people would want this option is to simulate TC game where TC is not possible (e.g. running many games in parallel).

The vast majority of users would not need to run non-TC games.

Maybe the alternative would be to have the opposite (playouts by default, visits through the hidden flag), but I think this needs some discussion in #dev-public as maybe it will break something. E.g. in selfplay it indeed will change the behaviour but there we can have a different default for now..

Reply:
Here are my thoughts:

- The default behavior should be kept as visits because visits = nodes is a common perception. If we changed that behavior it would probably leave more people confused. Like you said, only a few people need this flag, and having playouts by default could cause some issues in the training pipeline.
- A majority of the test results in #test-discuss are fixed-nodes tests. Fixing nodes or playouts has the advantage that you don't have to worry about weakening lc0 while you're playing a game or watching a stream or something while testing. I'm fairly confident this is the main reason behind most test results not being TC-based tests. 
- Most UCI settings aren't likely to be touched by the user at all unless they're trying to tune or experiment with it. I'd say this flag is more significant and more likely to be used than a lot of other options. The consequences of using playouts instead of visits in testing are decently important and I think this warrants the flag's inclusion.

Reply:
The polling in #dev-log has stagnated. It should be good to go now.